### ISSUE TITLE
训练语义分割模型，自动下载deeplabv3预训练模型DeepLabv3p_Xception65_COCO后，但报错不是有效参数。

### ISSUE THREAD

**Issue by panicwq** [PaddlePaddle/Paddle#38704]
Exception: File output/meter_seg\pretrain\DeepLabv3p_Xception65_COCO is not a paddle parameter file
- 版本、环境信息：
    1）PaddlePaddle版本：paddlepaddle-gpu   2.2.1
    2）Paddlex: 1.3.10
    3）系统环境：win10 python3.8
    4) GPU:GTX1650Ti  CUDA10.2 cudnn 8.2.2.26
- 训练信息
    1）单机，单卡

**Comment by cxxly** [comment]
您好，辛苦提供下复现该问题最小代码。

**Comment by panicwq** [comment]
代码就是paddlex官方案例：工业指针表计读数中训练语义分割模型的代码 train_segmentation.py
`import os

os.environ['CUDA_VISIBLE_DEVICES'] = '0'

import paddlex as pdx
from paddlex.seg import transforms


meter_seg_dataset = 'https://bj.bcebos.com/paddlex/examples/meter_reader/datasets/meter_seg.tar.gz'
pdx.utils.download_and_decompress(meter_seg_dataset, path='./')


train_transforms = transforms.Compose([
    transforms.Resize([512, 512]),
    transforms.RandomHorizontalFlip(prob=0.5),
    transforms.Normalize(),
])

eval_transforms = transforms.Compose([
    transforms.Resize([512, 512]),
    transforms.Normalize(),
])

train_dataset = pdx.datasets.SegDataset(
    data_dir='meter_seg/',
    file_list='meter_seg/train.txt',
    label_list='meter_seg/labels.txt',
    transforms=train_transforms,
    shuffle=True)
eval_dataset = pdx.datasets.SegDataset(
    data_dir='meter_seg/',
    file_list='meter_seg/val.txt',
    label_list='meter_seg/labels.txt',
    transforms=eval_transforms)

model = pdx.seg.DeepLabv3p(
    num_classes=len(train_dataset.labels), backbone='Xception65')
model.train(
    num_epochs=20,
    train_dataset=train_dataset,
    train_batch_size=4,
    eval_dataset=eval_dataset,
    learning_rate=0.1,
    pretrain_weights='COCO',
    save_interval_epochs=5,
    save_dir='output/meter_seg',
    use_vdl=True)`



> 你好，辛苦提供下复现该问题最小代码。

**Comment by cxxly** [comment]
@FlyingQianMM

**Comment by FlyingQianMM** [comment]
该问题用户已在PaddleX github提了相同的issue，https://github.com/PaddlePaddle/PaddleX/issues/1334 。
问题解答将直接至https://github.com/PaddlePaddle/PaddleX/issues/1334 进行
